plugins/dot/src/main/java/org/qbicc/plugin/dot/DotNodeVisitor.java

```diff
@@ -312,17 +306,194 @@ public String visit(final Appendable param, final StaticField node) {
         return name;
     }
 
-    public String visit(final Appendable param, final StaticInvocation node) {
+    // value handles
+
+    public String visit(Appendable param, ConstructorElementHandle node) {
+        String name = register(node);
+        appendTo(param, name);
+        nl(param);
+        attr(param, "label", "constructor\\n" + node.getExecutable());
+        nl(param);
```

Comment:
I guess it now needs `addEdge(param, node, node.getInstance(), EdgeType.VALUE_DEPENDENCY);` as well.